src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Controls/Label.cs

```diff
@@ -80,6 +80,29 @@ public Label() : base()
 
         #region Public Properties
 
+        /// <summary>
+        /// DependencyProperty for RecognizesAccessKey
+        /// </summary>
+        public static readonly DependencyProperty RecognizesAccessKeyProperty =
+            DependencyProperty.Register(
+                "RecognizesAccessKey",
+                typeof(bool),
+                typeof(Label),
+                new FrameworkPropertyMetadata(true));
```

Comment:
I do not think the default value should be true.

Reply:
I put it to true for backwards compatibility. Am I missing something?

Reply:
Correct, not being it `true` by default would be a serious breaking change.

Any reason for not using `ContentPresenter.RecognizesAccessKeyProperty.AddOwner` instead of declaring a new one?